src/actions/index.js

```diff
@@ -53,6 +53,22 @@ export const updateGIFProgress = progress => ({
   payload: { progress }
 });
 
+export const updateText = text => ({
+  type: types.UPDATE_TEXT,
+  payload: { text }
+});
+
+export const updateTextColor = fontColor => ({
+  type: types.UPDATE_TEXT_COLOR,
+  payload: { fontColor }
+});
+export const updateGIFFileName = name => {
```

Comment:
linebreak

Reply:
also i'd call the parameter `gifFileName` so that you can use object shorthand notation on the line with the payload.